### ISSUE TITLE
"Reload Metadata" crashes Tauon when the files aren't there anymore

### ISSUE THREAD

**Issue by Bauke** [Taiko2k/TauonMusicBox#477]
![image](https://user-images.githubusercontent.com/19501722/109571379-e999b500-7aeb-11eb-847e-1e09e073b6fe.png)

When the files are missing and you try to "Reload Metadata" on the album, Tauon crashes with the following error:

```
Error loading directroy
Reloading Metadata for Make-up Shadow.opus
Traceback (most recent call last):
  File "./tauon.py", line 43052, in <module>
    instance.render()
  File "./tauon.py", line 12725, in render
    self.items[to_call][2](self.reference)
  File "./tauon.py", line 18554, in reload_metadata
    pctl.master_library[track.index] = tag_scan(track)
  File "./tauon.py", line 4042, in tag_scan
    nt.modified_time = os.path.getmtime(nt.fullpath)
  File "/usr/lib/python3.8/genericpath.py", line 55, in getmtime
    return os.stat(filename).st_mtime
FileNotFoundError: [Errno 2] No such file or directory: '/media/bauke/Data/Beets/Library/Non-Album/ヨルシカ/Make-up Shadow.opus'
```

**Comment by Taiko2k** [comment]
This should be fixed in the new release (v6.5.4)

**Comment by Bauke** [comment]
Seems good now! :+1:
